docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,135 @@
+
+### TLS defeats passive and active attacks
+
+Transport Layer Security allows your client to verify that it's talking to the
+real Conjur server, and it uses standard secure technology to encrypt your
+secrets in transit. This means:
+
+* Your Conjur server will be `https:` instead of `http:`, just like a secure
+  website
+* Because the client knows it's talking to the real server, the "man in the
+  middle" will be exposed and the client won't leak any secret information
+* Since the traffic to and from Conjur is scrambled using secure encryption,
+  passive listeners on your network can't learn anything about the contents of
+  your secrets
+
+For these reasons, it is crucial to set up TLS correctly when you deploy Conjur.
+
+### Protect Your Secrets
+
+Do not leave this to chance: even a small flaw can totally compromise your
+secrets. Setting up Conjur and TLS without the appropriate expertise is like
+packing your own parachute and jumping out of a plane.
+
+That doesn't mean you should close the tab and walk away. It means you should get
+in touch with us and your own security team so we can ensure that you can employ
+Conjur successfully.
+
+[authn]: https://www.conjur.org/api.html#authentication-authenticate-post
+[secret-get]: https://www.conjur.org/api.html#secrets-retrieve-a-secret-get
+
+## Using NGINX To Proxy Traffic With TLS To Conjur
+
+[NGINX][NGINX] is a web proxy that's Free Software and easy to configure. This
+tutorial will show you how to use Docker to install Conjur and NGINX and
+configure them to provide Conjur with TLS.
+
+### Prerequisites
+
+This tutorial requires Docker and a terminal application. Prepare by following
+the prerequisite instructions found on [Install Conjur][prerequisites].
+
+Additionally, you will need the tutorial files from the Conjur source code
+repository. Here's how you get them:
+
+1. Install Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+1. Clone the Conjur repository
+   
+   In your terminal application, run:
+   ```sh-session
+   $ git clone git@github.com:cyberark/conjur.git
+   ```
+   This will create a folder called `conjur` in the current directory.
+
+### The Good Part
+
+The tutorial script will install Conjur and NGINX, configure them to work
+together, and connect a Conjur client to the running server. This is a full
+end-to-end working installation to allow you to see how the pieces connect
+together.
+
+```sh-session
+$ # start the Conjur+NGINX tutorial servers
+$ cd conjur/docs/tutorials/integrations/nginx
+$ ./start.sh
+```
+
+Take a look at the logs for the Conjur or NGINX servers:
+
+```sh-session
+$ docker-compose logs conjur
+$ docker-compose logs proxy
+```
+
+Run some commands in the Conjur client:
+
+```sh-session
+$ docker-compose exec client bash
+# conjur authn whoami
+# conjur list
+```
+
+These files show how the proxy setup works:
+
+* `docker-compose.yml` declares services and the connections between them
```

Comment:
Maybe talk a little bit here about how this file sets up the links / how the services are linked together?

Reply:
Also, the version of postgres that you use leads to a warning in the docker logs for the conjur server:

```
conjur_1    | .......SEQUEL DEPRECATION WARNING: Calling a dataset filtering method with multiple arguments or an array where the first argument/element is a string is deprecated and will be removed in Sequel 5.  Use Sequel.lit("account(resource_id) = ?", "test") to create an SQL fragment expression and pass that to the dataset filtering method, or use the auto_literal_strings extension.
conjur_1    | /var/lib/gems/2.2.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:1293:in `filter_expr'
conjur_1    | /var/lib/gems/2.2.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:1249:in `add_filter'
conjur_1    | /var/lib/gems/2.2.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:1034:in `where'
conjur_1    | /opt/conjur-server/app/models/resource.rb:70:in `search'
conjur_1    | /var/lib/gems/2.2.0/gems/sequel-4.46.0/lib/sequel/model/base.rb:1058:in `search'
conjur_1    | /opt/conjur-server/app/controllers/resources_controller.rb:14:in `index'
conjur_1    | /var/lib/gems/2.2.0/gems/actionpack-4.2.8/lib/abstract_controller/base.rb:198:in `process_action'
conjur_1    | /var/lib/gems/2.2.0/gems/actionpack-4.2.8/lib/action_controller/metal/rendering.rb:10:in `process_action'
conjur_1    | /var/lib/gems/2.2.0/gems/actionpack-4.2.8/lib/abstract_controller/callbacks.rb:20:in `block in process_action'
conjur_1    | /var/lib/gems/2.2.0/gems/activesupport-4.2.8/lib/active_support/callbacks.rb:117:in `call'
```

maybe it'd be worth using a different version just to avoid this bit of ugliness (if possible)?